joomla/joomla-cms
# Module Articles show only articles from current user

Pull Request for Issue # .

### Summary of Changes
With https://github.com/joomla/joomla-cms/pull/43738 a new articles module is added. This new module mod_articles module has no option to show only the articles from the current user. This PR adds that option.

### Testing Instructions
- Login in the frontend as a user who is author of some articles
- In the module set the Filtering Option "Author Filtering Type" to: Only from current user,
- See if only articles created by the logged in Author are shown.

### Actual result BEFORE applying this Pull Request
No option to show only articles form the current logged in user.

### Expected result AFTER applying this Pull Request
The option to show only articles form the current logged in user is added.

### Link to documentations
Please select:
